src/urllib3/contrib/pyopenssl.py

```diff
@@ -279,7 +279,7 @@ def recv(self, *args, **kwargs):
             if self.suppress_ragged_eofs and e.args == (-1, "Unexpected EOF"):
                 return b""
             else:
-                raise SocketError(str(e))
+                raise SocketError(e.args[0], str(e))
```

Comment:
SysCallError is always construct with a (errnum, message) tuple
https://www.pyopenssl.org/en/stable/api/ssl.html#OpenSSL.SSL.SysCallError